nonce.go

```diff
@@ -72,10 +72,13 @@ func nonceForChunkSecretBox(i encryptionBlockNumber) Nonce {
 // above. There's almost certainly no harm in using the same nonces here as
 // above, since the encryption keys are ephemeral and the signatures already
 // have their own context, but at the same time it's a good practice.
-func nonceForChunkSigncryption(i encryptionBlockNumber) Nonce {
+func nonceForChunkSigncryption(i encryptionBlockNumber, isFinal bool) Nonce {
```

Comment:
Added. Patterned after `nonceForMACKeyBoxV2`, lmk if I should do it differently. (Do we want part of it to be a constant string?)